Sources/CrossmintClient/CrossmintClientSDK.swift

```diff
@@ -15,16 +15,15 @@ public final class CrossmintClientSDK: ClientSDK, Sendable {
     init(apiKey: ApiKey, authManager: AuthManager? = nil) {
         self.apiKey = apiKey
 
-        if let bundleId = Bundle.main.bundleIdentifier {
-            secureStorage = KeychainSecureStorage(bundleId: bundleId)
-            secureWalletStorage = KeychainSecureWalletStorage(bundleId: bundleId)
-            crossmintService = DefaultCrossmintService(apiKey: apiKey, appIdentifier: bundleId)
-        } else {
-            secureStorage = NoOpSecureStorage()
-            secureWalletStorage = NoOpSecureWalletStorage()
-            crossmintService = NoOpCrossmintService()
+        guard let bundleId = Bundle.main.bundleIdentifier else {
+            Logger.sdk.error("Bundle identifier is required for Crossmint SDK to function properly")
```

Comment:
This is a great error message for the developer, but for our internal logs I think we can do this:
```suggestion
            Logger.sdk.error("Bundle.main.bundleIdentifier returned nil")
```